src/autocomplete/src/Autocomplete.js

```diff
@@ -75,17 +75,18 @@ const AutocompleteItems = ({
             const item = items[index]
             const itemString = itemToString(item)
 
-            return renderItem(
-              getItemProps({
+            return renderItem({
```

Comment:
I'm actually a bit confused about the type here. I noticed that `renderItem` is a property of type any.

Is this meant to be a React component? I realize this is out of scope for this story, but I think it might provide clarity if it is meant to be a React component to render it as a component would be rendered. Have you considered something like this?

```tsx
<RenderItem {...getItemProps({
  item,
  key: itemString,
  index,
  style,
  children: itemString,
  isSelected: itemToString(selectedItem) === itemString,
  isHighlighted: highlightedIndex === index
})} item={item} />
```

Of course this would require you to create an alias from `renderItem: RenderItem` when destructuring the properties on line 43.